src/webui/www/private/views/rss.html

```diff
@@ -499,7 +504,8 @@
                         match = true;
                         for (let i = 0; i < flattenedResp.length; ++i) {
                             if (((flattenedResp[i].uid ? flattenedResp[i].uid : "") !== rssFeedRows[i + 1].full_data.dataUid)
-                                || (flattenedResp[i].fullName !== rssFeedRows[i + 1].full_data.dataPath)) {
+                                || (flattenedResp[i].fullName !== rssFeedRows[i + 1].full_data.dataPath)
+                                || (flattenedResp[i].url !== rssFeedRows[i + 1].full_data.dataUrl)) {
```

Comment:
There is one problem here - folders do not contain url key, so feed list will be infinitely refreshed if any folders are present.

Reply:
I didn't test with folders. Will try this out and put up a fix.